01_node_intro/library/person.js

```diff
@@ -0,0 +1,21 @@
+class Person {
+    
+    /**
+     * @param {String} name 
+     * @param {String} city 
+     */
+    constructor(name = '', city = ''){
+        this._name = name;
```

Comment:
In general, it's not necessary to underline properties in JS. It might be confusing for code readers